src/app/GitUI/ScriptsEngine/ScriptInfo.cs

```diff
@@ -1,8 +1,13 @@
-﻿namespace GitUI.ScriptsEngine
+﻿using System.Text.RegularExpressions;
+
+namespace GitUI.ScriptsEngine
 {
     // WARNING: This class is serialized to XML!
-    public class ScriptInfo
+    public partial class ScriptInfo
     {
+        [GeneratedRegex("&(?!&)")]
```

Comment:
```suggestion
        // Match a single & (lookahead to not be followed by a second &)
        [GeneratedRegex("&(?!&)")]
```

Reply:
For lookahead it may be good to explain. Skip it if you prefer

Reply:
I was going to ask for the comment as well...